src/consensus/pbft/pbft.h

```diff
@@ -67,6 +67,65 @@ namespace pbft
 
   class PbftEnclaveNetwork : public INetwork
   {
+  private:
+    void serialize_message(uint8_t*& data, size_t& size, Message* msg)
```

Comment:
Why is this function needed? it is only called once?

Reply:
I will use it in the next PR which will do encryption of some pbft messages